bcda/service/service.go

```diff
@@ -496,8 +489,8 @@ func (s *service) setClaimsDate(args *models.JobEnqueueArgs, conditions RequestC
 		args.ClaimsWindow.UpperBound = conditions.claimsDate
 	}
 
-	for pattern, cfg := range s.acoConfig {
-		if pattern.MatchString(conditions.CMSID) {
+	for _, cfg := range s.acoConfigs {
+		if cfg.patternExp.MatchString(conditions.CMSID) {
 			args.ClaimsWindow.LowerBound = cfg.LookbackTime()
 			break
 		}
```

Comment:
It would seem this is config order dependent?  i.e. if two models overlap (such as with a pilot) it will just use whichever is found first.  Do we need to make a special note of that somewhere?

Reply:
Yeah that's correct, I'll look to see if there is a confluence spec somewhere on the ACO config file, but maybe we should also a) document it in the yaml and b) document it in the ACOConfig struct.   Thoughts?

Reply:
I have added comments in the struct and in each place the configs are searched.